packages/engine/src/avatar/components/AvatarControllerComponent.ts

```diff
@@ -26,6 +26,8 @@ export type AvatarControllerComponentType = {
   /** @todo refactor animation system */
   speedVelocity: { value: number }
   translationApplied: Vector3
+  // Below are autopilot input values
+  autopilotWalkpoint: Vector3 | undefined
```

Comment:
let's call this autopilotTarget and have it stored in hyperflux rather than the controller component, as it will only ever be used by the local user's avatar